brain/compartmentReportView.cpp

```diff
@@ -119,10 +55,30 @@ const CompartmentReportMapping& CompartmentReportView::getMapping() const
     return _impl->mapping;
 }
 
+// using float is necessary to match brion::CompartmentReport
+// frame index calculation
+inline size_t getFrameIndex(float t, float start, float timestep)
```

Comment:
Put inside an anonymous namespace and prepend a _ to the name to stress it's a private function.